android/src/main/java/io/github/elyx0/reactnativedocumentpicker/PathResolver.java

```diff
@@ -0,0 +1,245 @@
+package io.github.elyx0.reactnativedocumentpicker;
+
+import android.annotation.TargetApi;
+import android.content.ContentResolver;
+import android.content.ContentUris;
+import android.content.Context;
+import android.database.Cursor;
+import android.net.Uri;
+import android.os.Build;
+import android.os.Environment;
+import android.provider.DocumentsContract;
+import android.provider.MediaStore;
+import android.text.TextUtils;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.InputStream;
+
+/**
+ * Original file from rn-fetch-blob
+ */
+public class PathResolver {
+    @TargetApi(19)
+    public static String getRealPathFromURI(final Context context, final Uri uri) {
```

Comment:
would you mind replacing all of this file with this solution?

https://stackoverflow.com/a/52814247/2070942

thanks!